compiler/rustc_session/src/options.rs

```diff
@@ -989,7 +989,7 @@ mod parse {
             Some("next") => *slot = TraitSolver::Next,
             Some("next-coherence") => *slot = TraitSolver::NextCoherence,
             // default trait solver is subject to change..
-            Some("default") => *slot = TraitSolver::Classic,
+            Some("default") => *slot = TraitSolver::NextCoherence,
```

Comment:
how does this work :sweat_smile: don't you also have to change the default value in case no argument is present?

Reply:
ah yes `parse_trait_solver` would only be called when there is some cli arg to parse, and otherwise use `TraitSolver::Classic` as the default value.

Reply:
💀